yarn.lock

```diff
@@ -1343,9 +1343,9 @@
     glob "^7.2.0"
 
 "@hypothesis/frontend-shared@^6.1.1":
-  version "6.1.1"
-  resolved "https://registry.yarnpkg.com/@hypothesis/frontend-shared/-/frontend-shared-6.1.1.tgz#c3bd46f162e20c9c5612dd66f3fa4bb2e911f674"
-  integrity sha512-GGk6eb8pRNuqLN+QtQHQn+CC2ab7m5+a6cye0MSWkvuEclh8wLSxEHQk8/74xN9BJeWRWPbBNEswxKui8kph/A==
+  version "6.3.0"
```

Comment:
I needed a newer version to get some `Button` and `IconButton` styling-API updates.

Even with dependabot running on a monthly cadence in this project, this package seemed more out-of-date than I would have expected.